### PR TITLE
client for connection invitation management

### PR DESCRIPTION
**Pull request by wbaccinelli**
Part of the solution to https://github.com/PerfectFit-project/virtual-coach-issues/issues/248.

These are functions in the client to use the  endpoints developed in [this PR](https://github.com/PerfectFit-project/niceday-components/pull/14).

### CODE REVIEW

File: niceday_client/niceday_client.py [hunk 1]
```diff
@@ -268,3 +268,29 @@ def upload_file(self, user_id: int, filepath: str, file):
...
+
+    def get_invitation_requests(self):
+        """
+        Get the list of pending connection invitations of a therapist
+
+        """
+        url = self._niceday_api_uri + 'connectionrequests'
+        query_response = self._call_api('GET', url)
+        # convert the json response into a list of dict
+        response_json = json.loads(query_response.content)
```
Comment (raar1) [inline niceday_client/niceday_client.py:280]: I suppose you could add a check to see that the dict contains the expected keys for this request. Or does something else do this later? I suppose you still want to let through error responses, so maybe this is fine.

**Review by raar1 — APPROVED** [review]
Looks good. Only wondering about validation of responses but maybe that doesn't go here anyway.

File: niceday_client/niceday_client.py (hunk 1 above)
Comment (wbaccinelli) [reply niceday_client/niceday_client.py:280]: Yes, this part should be transparent. But actually this verification has to be added to the api endpoint as a descriptor of the success response. I am going to do this.

Comment (raar1) [reply niceday_client/niceday_client.py:280]: Ah yes, that would be a much better place to put it

**Pull request merged**